app/src/main/java/org/xbmc/kore/service/library/SyncUtils.java

```diff
@@ -460,26 +460,24 @@ public static void disconnectFromLibrarySyncService(Context context, ServiceConn
      * Checks if a running LibrarySyncService is syncing any given SyncType from a specific host
      * @param service running LibrarySyncService. Use {@link #connectToLibrarySyncService(Context, OnServiceListener)} to connect to a running LibrarySyncService
      * @param hostInfo host to check for sync items currently running or queued
-     * @param syncTypes sync types to check
-     * @return true if any of the given syncTypes is running or queued, false otherwise
+     * @param syncType sync type to check
+     * @return The first {@link SyncItem} of the given syncType if any is running or queued, null otherwise
      */
-    public static boolean isLibrarySyncing(LibrarySyncService service, HostInfo hostInfo, String... syncTypes) {
-        if (service == null || hostInfo == null || syncTypes == null)
-            return false;
+    public static SyncItem isLibrarySyncing(LibrarySyncService service, HostInfo hostInfo, String syncType) {
```

Comment:
We should rename this method to `getSyncItem` as it is convention that `is*` methods return true or false.